packages/components/app/styles/components/table.scss

```diff
@@ -119,23 +119,31 @@ $hds-table-cell-padding-tall: 20px 16px;
 }
 
 // TABLE DENSITY (TBODY ROW HEIGHT)
+.hds-table--density-short .hds-table__tbody th,
```

Comment:
@MelSumner can you please move these changes to a separate PR, so they're isolated? Thanks

Reply:
Sure! I thought from your comments elsewhere that you would want them combined. Sorry about that!